config/clusters/cloudbank/avc.values.yaml

```diff
@@ -19,7 +19,28 @@ jupyterhub:
           name: CloudBank
           url: http://cloudbank.org/
   hub:
+    loadRoles:
+      # Should use this, not hub.config.JupyterHub.load_roles - that will
+      # override any existing load_roles set by z2jh
+      section1-student:
+        scopes: [read:groups!group=section1]
+        groups: [section1]
+      section2-student:
+        scopes: [read:groups!group=section2]
+        groups: [section2]
+      section1-teacher:
+        scopes: [read:users!group=section1, servers!group=section2]
+        users: [sean.smorris@gmail.com]
     config:
+      loadGroups:
```

Comment:
This is jupyterhub (the software) native config, so it should be load_groups. The helm chart use camelCase for things while the jupyterhub python native config is camel_case